components/Common/Blockquote/index.tsx

```diff
@@ -0,0 +1,16 @@
+import type { ComponentProps, FC, PropsWithChildren } from 'react';
+
+import styles from './index.module.css';
+
+type BlockquoteProps = ComponentProps<'blockquote'>;
+
+const Blockquote: FC<PropsWithChildren<BlockquoteProps>> = ({
+  children,
+  cite,
```

Comment:
What exactly is this cite prop? Is this part of the native Blockquote HTML element?

Reply:
Yes, it's an attribute of the Blockquote HTML element. I couldn't figure out exactly what it does. It seems to have only a semantic effect 😄